src/components/layout/footer.tsx

```diff
@@ -4,34 +4,81 @@ import copyrightLogo from "@public/images/icons/copyright.svg";
 import githubIcon from "@public/images/icons/github.svg";
 import linkedinIcon from "@public/images/icons/linkedin.svg";
 import slackIcon from "@public/images/icons/slack.svg";
+import youtubeIcon from "@public/images/icons/youtube.svg";
+
 import Image, { StaticImageData } from "next/image";
 import React from "react";
+import { Input } from "../ui/input";
+import { Button } from "../ui/buttons/button";
 
 // SocialLink Component
-const SocialLink = ({ href, src, alt, ariaLabel }: { href: string; src: StaticImageData; alt: string; ariaLabel: string }) => (
-  <a href={href} target="_blank" rel="noopener noreferrer" className="mr-4" aria-label={ariaLabel}>
+const SocialLink = ({
+  href,
+  src,
+  alt,
+  ariaLabel,
+}: {
+  href: string;
+  src: StaticImageData;
```

Comment:
better to create an interface or type separately, that'd look cleaner and easy to use